lib/src/summarize.dart

```diff
@@ -4,12 +4,51 @@
 
 library dart_pad.summarize;
 
+import '../services/dartservices.dart';
 /// Instances of this class take string input of dart code and output a text 
 /// description ofthe code's size, packages, and other useful information
 class Summarizer {
   
-  String summarize(String input) {
-    return "foo";
+  String summarize(SourceRequest input, AnalysisResults result) {
```

Comment:
Let's change the first parameter to `String sourceCode`, and name the 2nd param `analysis` or `analysisResults`. `result` as a param name is a bit too generic.